doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -365,19 +396,27 @@ To enable Rivest-Shamir-Adleman (RSA) support, set one or more of the following
 RSA driver configurations
 =========================
 
-You can use the following Kconfig options for fine-grained control over which drivers provide RSA support:
+You can use the following table to check the RSA algorithm support of each driver:
+
++-----------------------+---------------------------+----------------------------+
+| RSA algorithms        | nrf_cc3xx driver support  | nrf_oberon driver support  |
++=======================+===========================+============================+
+| RSA OAEP              | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+| RSA PKCS#1 v1.5 crypt | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+| RSA PKCS#1 v1.5 sign  | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+| RSA PSS               | Not supported             | Supported                  |
++-----------------------+---------------------------+----------------------------+
+
+The option :kconfig:option:`PSA_USE_CC3XX_SIGNATURE_DRIVER` enables the driver :ref:`nrf_security_drivers_cc3xx` for the RSA PKCS#1 v1.5 signing algorithm.
+
+The option :kconfig:option:`PSA_USE_CC3XX_ASYMMETRIC_DRIVER` enables the driver :ref:`nrf_security_drivers_cc3xx` for RSA PKCS#1 v1.5  and RSA OAEP encryption.
```

Comment:
Our docs should correspond to the PSA Crypto docs.

Also, it is anomalous for most chapters to describe one option, and this chapter to describe to options.

If we align to the PSA Crypto docs the chapters will be called "Asymmetric signature", and "Asymmetric encryption" and have one Kconfig option each.

This is a pre-existing problem so we can create a task for this instead of fixing it here if you prefer.

Reply:
I fully agree with this statement and I will create a task for this. I noticed that as well but I didn't want to completely rewrite the documentation since this will be a separate task altogether indeed. 
See: NCSDK-22757

Reply:
Thank you very much